File: ASCS-Vehicle/src/team/hobbyrobot/tdn/base/TDNParsers.java
package team.hobbyrobot.tdn.base;

import team.hobbyrobot.tdn.core.TDNRootParser;
import team.hobbyrobot.tdn.core.TDNTypeParser;

public final class TDNParsers
{
	public static final TDNTypeParser ARRAY = new ArrayParser();
	public static final TDNTypeParser BOOLEAN = new BooleanParser();
	public static final TDNTypeParser FLOAT = new FloatParser();
	public static final TDNTypeParser INTEGER = new IntegerParser();
	public static final TDNTypeParser STRING = new StringParser();
	public static final TDNTypeParser ROOT = new TDNRootParser();
}

File: TransmissionDataNotation/Base/TDNParsers.cs
using System;
using System.Collections.Generic;
using System.Text;
using Team.HobbyRobot.TDN.Core;

namespace Team.HobbyRobot.TDN.Base
{
    public static class TDNParsers
    {
		public static readonly ITDNTypeParser ARRAY = new ArrayParser();
		public static readonly ITDNTypeParser BOOLEAN = new BooleanParser();
		public static readonly ITDNTypeParser FLOAT = new FloatParser();
		public static readonly ITDNTypeParser INTEGER = new IntegerParser();
		public static readonly ITDNTypeParser STRING = new StringParser();
		public static readonly ITDNTypeParser ROOT = new TDNRootParser();
	}
}
